src/prompt_based/model.py

```diff
@@ -62,7 +62,11 @@ def normalize(self,code):
         Returns:
             str: refined version of the code
         """
-        return unicodedata.normalize('NFKD', code).encode('ascii', 'ignore').decode('utf-8')
+        return (
+            unicodedata.normalize("NFKD", code)
+            .encode("ascii", "ignore")
+            .decode("utf-8")
+        )
```

Comment:
To make reviews easier for the maintainer, always try to keep their formatting, especially if no code has changed. Prevents them spending time looking for differences where there are none